apps/condo/domains/billing/schema/AllResidentBillingReceiptsService.test.js

```diff
@@ -348,6 +348,47 @@ describe('AllResidentBillingReceiptsService', () => {
             expect(Big(receiptAfterPayment.paid).toFixed(2)).toEqual(paidAmounts.reduce((sum, amount) => Big(sum).add(Big(amount)).toFixed(2), '0.00'))
             expect(receiptAfterPayment.isPayable).toBeTruthy()
         })
+
+        test('paid field calculated when organization bank info changed', async () => {
```

Comment:
One test is not enought for this changes. You should test at least few negative cases.

Reply:
Changed most tests to use payment creation with connection to receipt, and 1 to say that we do not count payments for virtual receipts at all